tests/models/nn/transformers/test_base.py

```diff
@@ -152,10 +153,24 @@ def test_save_load_for_fitted_model(
 
     @pytest.mark.parametrize("test_dataset", ("dataset", "dataset_item_features"))
     @pytest.mark.parametrize("model_cls", (SASRecModel, BERT4RecModel))
+    @pytest.mark.parametrize("map_location", ("cpu", torch.device("cuda:0"), None))
+    @pytest.mark.parametrize(
+        "config_update",
+        (
+            {
+                "model_config": {
```

Comment:
`config_update` should not have "model_config" key, user should just be able top pass `config_update={"get_val_mask_func ": None}` 
Please update this logic everywhere